src/main/scala/main.scala

```diff
@@ -173,6 +187,22 @@ def setupUI(): Unit =
   filterText.appendChild(todayLabel)
   document.body.appendChild(showText)
 
+def addDiscrepancyPanel(discrepancies: Set[String]) =
+  val container = document.createElement("div").asInstanceOf[dom.html.Div]
+  container.id = "discrepancyPanel"
+  discrepancies.foreach(discrepancy => 
+    val entrySpan = document.createElement("span").asInstanceOf[dom.html.Span]
+    entrySpan.innerHTML = discrepancy
+    container.appendChild(entrySpan)
+  )
+  document.body.prepend(container)
+
+def addTimeEditFailPanel() = 
+  val container = document.createElement("div").asInstanceOf[dom.html.Div]
+  container.id = "timeEditFailPanel"
+  container.innerHTML = "Data från TimeEdit kunde ej hämtas - Dubbelkolla din sal där"
```

Comment:
Suggestion for changed message that does not require an action by user that is difficult with no network: 
"TimeEdit i molnet svara inte just nu - KaptenAlloc kör med tidigare sparad data." 
(Or should we just be silent - it is not a bug to have flightmode on...? 
Or show a non-intrusive icon of newtwork on/off in the corner?)
But we can fix this later. OK as is for now.